_includes/footer.html

```diff
@@ -1,67 +1,103 @@
-                        <p><a href="https://github.com/vacp2p" target="_blank">Check out the code</a></p>
-                        <p>Specs, research, proof of concepts. Check it out!</p>
-                    </li>
-                    <li>
-                        <a href="https://forum.vac.dev/" target="_blank"><svg width="60" height="59" viewBox="0 0 60 59" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M31.0942 58.2734C23.9642 58.2734 17.4449 55.6679 12.4045 51.3578L0 53.0525L4.79223 41.2247C3.07059 37.4276 2.18189 33.3059 2.1859 29.1367C2.1859 13.0459 15.1286 0 31.0942 0C47.0586 0 60 13.0459 60 29.1367C60 45.2305 47.059 58.2734 31.0942 58.2734ZM46.8786 29.0534V28.9722C46.8786 20.5645 40.9494 14.5692 30.726 14.5692H19.6843V43.7059H30.5624C40.8652 43.7063 46.8786 37.4607 46.8786 29.0534ZM30.8478 36.5473H27.6181V21.7299H30.8478C35.5912 21.7299 38.7397 24.4342 38.7397 29.0983V29.1794C38.7397 33.8827 35.5912 36.5473 30.8478 36.5473Z" fill="#121212"/></svg></a>
-                        <p><a href="https://forum.vac.dev/" target="_blank">Join the research forum</a></p>
-                        <p>Ask questions, find answers and discuss recent research at our form</p>
-                    </li>
-                    <li>
-                        <a href="https://t.me/vacp2p" target="_blank"><svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg"> <path d="M30 60C46.5685 60 60 46.5685 60 30C60 13.4315 46.5685 0 30 0C13.4315 0 0 13.4315 0 30C0 46.5685 13.4315 60 30 60Z" fill="#121212"/> <path fill-rule="evenodd" clip-rule="evenodd" d="M12.4661 29.8691C21.6909 26.1517 27.8422 23.701 30.9201 22.5169C39.7079 19.1362 41.5339 18.5489 42.7241 18.5295C42.9859 18.5252 43.5712 18.5852 43.9503 18.8698C44.2704 19.1101 44.3585 19.4346 44.4007 19.6624C44.4428 19.8902 44.4953 20.4091 44.4536 20.8146C43.9774 25.4426 41.9168 36.6736 40.8685 41.857C40.4249 44.0503 39.5451 44.5197 38.6996 44.5917C36.8619 44.7481 35.4729 43.7344 33.6931 42.6553C30.908 40.9667 29.2236 40.1875 26.5202 38.5397C23.396 36.6355 26.1451 35.7451 27.9256 34.0346C28.3916 33.587 36.3528 26.4292 36.3185 26.2139C36.3061 26.1358 36.0699 25.364 35.8848 25.2119C35.6998 25.0597 35.4267 25.1118 35.2297 25.1531C34.9503 25.2118 30.5011 27.9317 21.8821 33.3131C20.6192 34.1152 19.4753 34.506 18.4505 34.4855C17.3206 34.4629 15.1472 33.8946 13.5316 33.4088C11.5499 32.813 10.8184 32.5324 10.9555 31.5206C11.0269 30.9935 11.5305 30.443 12.4661 29.8691V29.8691Z" fill="white"/> </svg></a>
-                        <p><a href="https://t.me/vacp2p" target="_blank">Chat with us</a></p>
-                        <p>Join the conversation with us on Telegram</p>
-                    </li>
-                </ul>
-            </div>
-        </div>
+<footer class="footer bg-black flex flex-shrink-0 justify-center">
+  <div
+    class="
+      container
+      max-w-screen-xl
+      flex
+      sl:justify-between
```

Comment:
Is this code formatting deliberate? Looks a bit odd with a lot of newlines

Reply:
Fixed my Prettier settings